src/terminalOperations.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const readline = require('readline');
+const { generateNumber } = require('./generatNumber');
+const { calculateBullsAndCows } = require('./calculateBullsAndCows');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const number = generateNumber();
+
+async function terminalOperations() {
+  terminal.question('Guess the number: ', answer => {
+    calculateBullsAndCows(number, answer);
```

Comment:
the program should check the entered number (if it has exactly 4 different digits)